jenkins/spark-premerge-build.sh

```diff
@@ -121,6 +121,13 @@ nvidia-smi
 
 . jenkins/version-def.sh
 
+BUILD_SNAPSHOTS="false"
+PREMERGE_PROFILES="-PnoSnapshots,pre-merge"
+if [[ ${PROJECT_VER} =~ ^22\.06\. ]]; then # enable snapshot builds for active development branch only
```

Comment:
The proposal to reuse auto-merge has the advantage is that it makes auto-merge config the single source of truth. And as part of our standard practice we never forget to update it .
 
With this solution there will be two places to update when the current branch enters the burn-down state. On the other hand it gives us the flexibility to manage auto-merge and BUILD_SHIMS independently but in a simplified form. I am not sure we have a need for this flexibility.

Reply:
> And as part of our standard practice we never forget to update it .

Actually version-def.sh is the a source of truth for testing. auto-merge BASE could probably be following other naming patterns that do not reflect versions